tests/acceptance-tests/src/acceptance/http_closed_test.go

```diff
@@ -0,0 +1,44 @@
+package acceptance_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+
+	"net"
+	"time"
+)
+
+var _ = Describe("Http is closed", func() {
+
+	var (
+		CONNETION_TIMEOUT = 11 * time.Second
```

Comment:
There's a typo here, `s/CONNETION/CONNECTION/`.

I wonder if the name `FIREWALL_TIMEOUT` is more descriptive though.